gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.2-bin.zip
```

Comment:
Why downgrade?

Reply:
I kept having errors while executing ```./gradlew generateContractWrappers``` due to the 7.0.2. version of gradle, so I changed it to the older version. I also peeked at how @domagojlatecki did it, and he is using 6.8.2 as well.

Reply:
Gradle likes to deprecate things and then remove them relatively soon, so there are still some plugins which are not compatible with `7.x`.